subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/ConfigurationDeprecatedExtensions.kt

```diff
@@ -278,7 +278,7 @@ val <T : Configuration> NamedDomainObjectProvider<T>.resolvedConfiguration
 /**
  * See [Configuration.getUploadTaskName].
  */
-@Suppress("DEPRECATION")
+@Suppress("DEPRECATION") // TODO:Finalize Upload Removal - Issue #21439
 @Deprecated(deprecationMessage, replaceWith = ReplaceWith("get().uploadTaskName"))
 val <T : Configuration> NamedDomainObjectProvider<T>.uploadTaskName
```

Comment:
I think we can just delete this now, the reason being that this isn't a exact reference to `Configuration.getUploadTaskName()`, but it's here to allow someone to use `getUploadTaskName` with a `Provider<Configuration>`.